GNSSLogger/app/src/main/java/com/google/android/apps/location/gps/gnsslogger/SensorFusionContainer.java

```diff
@@ -21,19 +21,21 @@
 import android.location.GnssNavigationMessage;
 import android.location.GnssStatus;
 import android.location.Location;
-import android.location.LocationListener;
```

Comment:
Why did you remove this import? If you restore it, you can remove the `android.location.` qualifier on LocationManager in the rest of the code.

Reply:
I removed this import because both android.location and com.google.android.gms.location have differing implementations of the LocationListener class used in this file. I figured using the full name for both classes throughout the file would keep things consistent since only one can be imported and make it easier to distinguish which LocationListener implementation was being referenced at any given time. Would you like me to put the import back for the android.location version and remove the qualifier throughout the rest of the file? Happy to do so if you think that makes more sense.

Reply:
Oh, no that makes sense. Thanks!